src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,31 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(actor);
+            transaction.commit();
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Cannot add actor to the DB", e);
```

Comment:
Message can be more clearer if you also showed what actor are this method trying to add.
You have same problem in all `DataProcessingException` messages.

Can be something like this I guess:
```suggestion
            throw new DataProcessingException("Cannot add actor to the DB: " + actor, e);
```